examples/cubic_planar.js

```diff
@@ -0,0 +1,170 @@
+
+const parent = new itowns.THREE.Mesh(
+    new itowns.THREE.BoxGeometry(8000, 8000, 8000),
+    new itowns.THREE.MeshBasicMaterial({ color: 0xdddddd} ));
+parent.scale.copy(scale);
+parent.updateMatrixWorld(true);
+
+view.scene.add(parent);
+
+for (let index=0; index<wmsLayers.length; index++) {
+    const wms = wmsLayers[index];
+    const obj = new itowns.THREE.Object3D();
+    // obj.position.add(cubeTransformations[index].position);
+    const offset = extent.center().xyz().negate().applyEuler(cubeTransformations[index].rotation);
+    offset.add(cubeTransformations[index].position.divide(scale));
+    obj.position.copy(offset);
+    obj.rotation.copy(cubeTransformations[index].rotation);
+    parent.add(obj);
+    obj.updateMatrixWorld(true);
+
+
+    const tileLayer = itowns.createPlanarLayer('planar' + wms + index, extent, { object3d: obj });
+    tileLayer.disableSkirt = true;
+
+    itowns.View.prototype.addLayer.call(view, tileLayer);
+
+    itowns.View.prototype.addLayer.call(view, {
+        update: itowns.updateLayeredMaterialNodeImagery,
+        url: 'https://download.data.grandlyon.com/wms/grandlyon',
+        networkOptions: { crossOrigin: 'anonymous' },
+        type: 'color',
+        protocol: 'wms',
+        version: '1.3.0',
+        id: 'wms_imagery' + wms + index,
+        name: wms,
+        projection: 'EPSG:3946',
+        transparent: false,
+        extent: extent,
+        bbox_url: 'wsen',
+        updateStrategy: {
+            type: 0,
+            options: {},
+        },
+        options: {
+            mimetype: 'image/jpeg',
+        },
+    }, tileLayer);
+
+    itowns.View.prototype.addLayer.call(view, {
+        update: itowns.updateLayeredMaterialNodeElevation,
+        url: 'https://download.data.grandlyon.com/wms/grandlyon',
+        type: 'elevation',
+        protocol: 'wms',
+        networkOptions: { crossOrigin: 'anonymous' },
+        version: '1.3.0',
+        id: 'wms_elevation' + wms + index,
+        name: 'MNT2012_Altitude_10m_CC46',
+        style: '',
+        projection: 'EPSG:3946',
+        transparent: false,
+        extent: extent,
+        bbox_url: 'wsen',
+        heightMapWidth: 256,
+        options: {
+            mimetype: 'image/jpeg',
+        },
+    }, tileLayer);
+
+    // Since the elevation layer use color textures, specify min/max z
+    tileLayer.materialOptions = {
+        useColorTextureElevation: true,
+        colorTextureElevationMinZ:  -600,
+        colorTextureElevationMaxZ: 400,
+    };
+}
+
+// Since PlanarView doesn't create default controls, we manipulate directly three.js camera
+// Position the camera at south-west corner
+c = new itowns.Coordinates('EPSG:3946', extent.west(), extent.south(), 2000);
+view.camera.camera3D.position.set(3, 2, 3);// copy(c.xyz().multiplyScalar(scale.x * 0.99));
+view.camera.camera3D.updateMatrixWorld(true);
+view.camera.camera3D.lookAt(new itowns.THREE.Vector3(0, 0, 0));
+
+const revolution = {};
```

Comment:
(the linter should warn you about this and arrow functions. If it does not, please tell me)